ray-curator/ray_curator/examples/audio/fleurs_and_wer_example.py

```diff
@@ -0,0 +1,78 @@
+from dataclasses import dataclass
+
+from loguru import logger
+from nemo.core.config import hydra_runner
+from omegaconf import OmegaConf
```

Comment:
Does this mean we should add these as dependencies?

Reply:
it is in nemo_toolkit[asr]'s dependencies